README.md

```diff
@@ -88,3 +88,33 @@ After you've started the dev server, you can open the interactive browser tests
 ```sh
 npm run cypress:open
 ```
+
+### Chore: Dependency Updates
+
+You can see which dependencies have new releases by first making sure your local dependencies are up-to-date by executing `npm ci` and then running `npm outdated`.
+The following dependencies should be ignored:
+
+- Wikit (i.e. `@wmde/wikit-tokens` and `@wmde/wikit-vue-components`):
+  we’re using a newer pre-release version and don’t want to downgrade to the latest full release.
+- Vue and Vuex:
+  in production, we use the versions shipped by MediaWiki core,
```

Comment:
Do we have a place that ought to show what version is shipped by MediaWiki core? If so, we may want to link that here

Reply:
+1 Sounds like a good idea!!

Reply:
We could link to [foreign-resources.yaml](https://gerrit.wikimedia.org/g/mediawiki/core/+/master/resources/lib/foreign-resources.yaml), but I’m not sure how helpful it is (might be a bit long / confusing)?

Reply:
Maybe with something like "The current core version is listed in [foreign-resources.yaml](https://gerrit.wikimedia.org/g/mediawiki/core/+/master/resources/lib/foreign-resources.yaml)."
Then it is implied that one has to look through a list of things.

Reply:
Done, thanks!